en/ios/objects.mdown

```diff
@@ -492,14 +492,26 @@ For more details on `PFQuery` please look at the query portion of this guide.  A
 
 ## Data Types
 
-So far we've used values with type `NSString`, `NSNumber`, and `%{ParseObject}`. Parse also supports `NSDate`, and `NSNull`.
+So far we've used values with type `NSString`, `NSNumber`, and `%{ParseObject}`. Parse also supports `NSDate`, `NSDictionary`, `NSArray`, and `NSNull`.
 
-You can nest `NSDictionary` and `NSArray` objects to store more structured data within a single `%{ParseObject}`.
+You can nest `NSDictionary` and `NSArray` objects to store more structured data within a single `%{ParseObject}`. Overall, the following types are allowed for each field in your object:
+
+* String
```

Comment:
If you're doing this in every platform, do you want to change this into `NSNumber`, `NSString`, `NSArray`....? You can also add token so we can do `%{String}`. IIRC, the logic behind token is "if the token is not found in config, then print the token itself. e.g. %{ParseFile} prints `NSString` if it exists in the config map, otherwise prints `String`)